src/v/debug_bundle/tests/json_test.cc

```diff
@@ -134,22 +139,20 @@ TYPED_TEST(JsonTypeTest, BasicType) {
         tmp.tm_isdst = -1;
         std::time_t tt = std::mktime(&tmp);
 
-        this->expected = debug_bundle_parameters{
-          .authn_options
-          = scram_creds{.username = security::credential_user{"user"}, .password = security::credential_password{"pass"}, .mechanism = "SCRAM-SHA-256"},
-          .controller_logs_size_limit_bytes = 42,
-          .cpu_profiler_wait_seconds = 42s,
-          .logs_since = special_date::yesterday,
-          .logs_size_limit_bytes = 42,
-          .logs_until = clock::from_time_t(tt),
-          .metrics_interval_seconds = 42s,
-          .partition = std::vector<partition_selection>{
-            {{model::ns{"foo"}, model::topic{"bar"}},
-             {{model::partition_id{1}, model::partition_id{2}}}},
-            {{model::kafka_namespace, model::topic{"baz"}},
-             {{model::partition_id{1},
-               model::partition_id{2},
-               model::partition_id{3}}}}}};
+        this->expected
+          = debug_bundle_parameters{
+            .authn_options
+            = scram_creds{.username = security::credential_user{"user"}, .password = security::credential_password{"pass"}, .mechanism = "SCRAM-SHA-256"},
+            .controller_logs_size_limit_bytes = 42,
+            .cpu_profiler_wait_seconds = 42s,
+            .logs_since = special_date::yesterday,
+            .logs_size_limit_bytes = 42,
+            .logs_until = clock::from_time_t(tt),
+            .metrics_interval_seconds = 42s,
+            .partition = std::vector<partition_selection>{{{model::ns{"foo"}, model::topic{"bar"}}, {{model::partition_id{1}, model::partition_id{2}}}}, {{model::kafka_namespace, model::topic{"baz"}}, {{model::partition_id{1}, model::partition_id{2}, model::partition_id{3}}}}},
+            .tls_enabled = true,
+            .tls_insecure_skip_verify = false,
+            .k8s_namespace = debug_bundle::k8s_namespace("k8s-namespace")};
```

Comment:
No idea why it changed so drastically... it passes clang format checks though :(

Reply:
I think if you always place a comma after the last designator you'll get more consistent formatting between changes. Probably would still result in a big diff here though 🤷